README.md

```diff
@@ -76,9 +81,9 @@ Below are the common issues you may face when building this repository's documen
     ```
   The table of contents is embedded into each topic with this template which increases the build time and HTML file sizes. Alternatively, you can override the browser's restrictions (which may be insecure). For example, Google Chrome and Microsoft Edge accept the `--allow-file-access-from-files` command line switch which allows loading local files.
```

Comment:
What does all this mean?

"which may be insecure" Do you mean unsecure?

Reply:
Yes, unsecure.